src/theme.js

```diff
@@ -12,18 +12,20 @@ const theme = createMuiTheme({
     }
   },
   typography: {
-    fontFamily: "Zinnebeeld",
+    fontFamily: "Open Sans",
```

Comment:
This should probably be `"Open Sans", Helvetica, Arial, sans-serif`. That way it'll fall back to a sans-serif font instead of the default serif one.

A tip - so you don't have to repeat yourself, you can define the font family above the theme:

```javascript
const fontFamily = '"Open Sans", Helvetica, Arial, sans-serif';
```

Then you can reference it in the theme object as `fontFamily`. For example, this line would be just `fontFamily: fontFamily,`.